platform-service/pyproject.toml

```diff
@@ -14,7 +14,7 @@ dependencies = [
     "cryptography>=41.0.7",
     "unstract-flags @ file:///${PROJECT_ROOT}/../unstract/flags",
     "requests>=2.31.0",
-    "unstract-sdk~=0.54.0rc1",
+    "unstract-sdk==0.54.0rc2",
```

Comment:
Dowe need to bump the rc versions in main?

Reply:
As per discussion with @hari-kuriakose resolving it.